Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -1190,22 +1189,62 @@ final class LayerTests: XCTestCase {
                      [ 0.074910110, 0.021107012, -0.049724963, -0.069670826],
                      [ 0.078670055, 0.022462710, -0.051899005, -0.075331904]],
                     accuracy: 1e-6)
+                let (𝛁lstm, _) = pullback(.init(inputs.map { LSTMCell<Float>.State(cell: $0, hidden: $0) }))
+                // TODO: Verify that LSTM gradients are correct using a reference implementation.
```

Comment:
I'm interested in doing this. Should we do something like check some TF python code into this repository that reproduces the weights?

Reply:
That sounds good!